src/state/store.js

```diff
@@ -5,7 +5,7 @@ import persistState from 'redux-localstorage';
 
 const localStorageStates = {
   locale: [],
-  permissions: ['loggedUser'],
+  permissions: ['loggedUser', 'loggedUserPemissionsLoaded'],
```

Comment:
1. `loggedUserPemissionsLoaded` - I'm supposed to find this in `state/permissions/reducer.js` but I can't find it there.
2. take note of its spelling